File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webapi/service/types/CallContext.java
package org.opcfoundation.webapi.service.types;

import org.eclipse.milo.opcua.stack.core.types.structured.CallMethodRequest;
import org.eclipse.milo.opcua.stack.core.types.structured.RequestHeader;
import org.springframework.lang.Nullable;
import org.springframework.web.context.request.NativeWebRequest;

import java.util.List;

public class CallContext extends ServiceContext {
    private final List<CallMethodRequest> methodsToCall;

    public CallContext(
            List<CallMethodRequest> methodsToCall,
            @Nullable String serverUri,
            @Nullable RequestHeader header,
            @Nullable NativeWebRequest webRequest)
    {
        super(serverUri,header,webRequest);
        this.methodsToCall = methodsToCall;
    }

    public List<CallMethodRequest> getMethodsToCall() {
        return methodsToCall;
    }
}

File: opcua-webservice-node/src/server/types/contexts/CallContext.ts
import { RequestHeader } from "opcua-webapi";
import { ServiceContext } from "./ServiceContext";
import { UaCallMethodRequest } from "opcua-webapi-ts";

export class CallContext extends ServiceContext
{
    private _methodsToCall: Array<UaCallMethodRequest>;
    
    constructor(
        methodsToCall: Array<UaCallMethodRequest>,
        serverUri?: string, 
        requestHeader? : RequestHeader)
    {
        super(serverUri, requestHeader);
        this._methodsToCall = (methodsToCall) ? methodsToCall : [];
    }

    get methodsToCall() : Array<UaCallMethodRequest>
    {
        return this._methodsToCall;
    }
}